edb/server/pgcluster.py

```diff
@@ -834,6 +834,7 @@ async def _get_cluster_type(
         managed_clouds = {
             'rds_superuser': RemoteCluster,    # Amazon RDS
             'cloudsqlsuperuser': RemoteCluster,    # GCP Cloud SQL
+            'azure_pg_admin': RemoteCluster,    # Azure Postgres
```

Comment:
Can you handle a possible KeyError (raise a nice error) when you're indexing this dict a few lines below?